src/mod_muc_sql.erl

```diff
@@ -245,6 +268,17 @@ set_affiliations(_ServerHost, _Room, _Host, _Affiliations) ->
 get_affiliation(_ServerHost, _Room, _Host, _LUser, _LServer) ->
     {error, not_implemented}.
 
+get_affiliations(_Host) ->
```

Comment:
Should this function hit the cache first?

Reply:
I suppose it could and fallback when it misses, but it will miss every time in the current design. This functions only used to get all enabled affiliations, which is only used once on startup. But if it was ever used in the future it'd be nice to hit the cache so I'll add that

Reply:
Actually I'm not sure right now how to easily do this with the ets_cache (lookup all key/value pairs) and since it would have no use case I'm going to skip this unless you feel its important